docker-compose.yml

```diff
@@ -730,16 +664,10 @@ services:
 ################################################################################
 # DELTA SUBSIDIES: START
 ################################################################################
-  delta-producer-background-jobs-initiator-subsidies:
+  delta-producer-background-jobs-initiator-generic:
```

Comment:
Two minor remarks but:
1. Wouldn't it make more sense to put this service in the `# DELTA GENERAL` section ?
2. Couldn't we just omit the `-generic` in the same ? Especially if it's in the DELTA GENERAL section, in speaks for itself ?